### ISSUE TITLE
How to implement model once pretrained for masked input sentences?

### ISSUE THREAD

**Issue by sfaux** [codertimo/BERT-pytorch#80]
I'm relatively new to pytorch and BERT, but I figured out how to pretrain the BERT model on my dataset. I want to use the model to predict the mask(s) in an input sentence. How do use the model to predict that?
